src/styles/main.scss

```diff
@@ -1,3 +1,70 @@
+$days: (mon 0,
+  tue 1,
+  wed 2,
+  thu 3,
+  fri 4,
+  sat 5,
+  sun 6);
+$month-length: 31;
+$width: 100px;
+$gap: 1px;
+$day-weeks: 7;
+
+html{
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
 body {
   margin: 0;
 }
+
+.calendar {
+  box-sizing: border-box;
+  height: 500px;
+  width: $day-weeks * $width + ($day-weeks - 1) * $gap;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+
+  &__day {
+    height: 98px;
+    width: 98px;
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    background-color: #eee;
+    border: 1px solid black;
+
+    transition: transform 500ms;
+
+
+    @for $i from 1 through $month-length {
+        &:nth-child(#{$i})::before {
+        content: '#{$i}';
+      }
+    }
+
+    &:hover {
+      background-color: #ffbfcb;
+      transform: translateY(-20px);
+      animation-duration: 0.5s;
```

Comment:
Pay attention to the difference between animation and transition
https://blog.hubspot.com/website/css-transition-vs-animation